Resources/config/config.yml

```diff
@@ -17,4 +17,4 @@ fos_elastica:
             types:
                 Title:
                     mappings:
-                        content: ~
\ No newline at end of file
+                        content: { type: 'string', boost: 5 }
```

Comment:
It'd be interesting for the future to document this somewhere.
If it is an arbitrary weighting, it is better to say it clearly.